### PR TITLE
release notes for v0.2.0

### PR DESCRIPTION
**Pull request by trz42**
WIP as more may be added.

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+v0.2.0 (26 November 2023)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Bug fixes:
+* none
+
+Improvements:
+* support for running tests inside the same job but after the build step (#222)
```
Comment (boegel) [inline RELEASE_NOTES:13]: We should mention `bot/test.sh` and `bot/check-test.sh` scripts that are picked up with the feature?

**Review by boegel — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES (hunk 1 above)
Comment (trz42) [reply RELEASE_NOTES:13]: Addressed in https://github.com/EESSI/eessi-bot-software-layer/pull/232/commits/c700a22c5b3db501b9bb58d18be0b37aa16b8182

File: RELEASE_NOTES [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+Bug fixes:
+* adds information on installing and using the smee client on `aarch64` (#233)
+
+Improvements:
+* support for running tests inside the same job but after the build step (#222)
+  * runs `bot/test.sh` and `bot/check-test.sh` if these are provided by the pull
+    request being processed
+  * adds a new setting (`job_test_unknown_fmt`) in the bot's configuration file
+* ensure the bot can build for both the EESSI pilot repository (`pilot.eessi-hpc.org`) and `software.eessi.io` (#229)
+* support specifying repository-specific buckets via `bucket_name` in configuration file (#230)
```
Comment (boegel) [inline RELEASE_NOTES:18]: should also mention PR #233

**Review by boegel — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+
+This is a minor release of the EESSI build-and-deploy bot.
+
+Bug fixes:
+* adds information on installing and using the smee client on `aarch64` (#233)
+
+Improvements:
+* support for running tests inside the same job but after the build step (#222)
+  * runs `bot/test.sh` and `bot/check-test.sh` if these are provided by the pull
+    request being processed
```
Comment (boegel) [inline RELEASE_NOTES:15]: I would rephrase this as:

```suggestion
  * runs `bot/test.sh` and `bot/check-test.sh` if these are provided by in the target repository
```

the fact that these scripts initially make part of the PR (or are updated in a PR) is a detail we don't need to mention here imho (since that applies to other stuff the bot picks up on in `bot/` too)

**Review by boegel — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES (hunk 2 above)
Comment (trz42) [reply RELEASE_NOTES:18]: Added it to "Bug fixes" in https://github.com/EESSI/eessi-bot-software-layer/pull/232/commits/c700a22c5b3db501b9bb58d18be0b37aa16b8182

File: RELEASE_NOTES (hunk 3 above)
Comment (trz42) [reply RELEASE_NOTES:15]: Yes. It was an attempt to distinguish a bit from the "repository" later mentioned.

Comment (trz42) [reply RELEASE_NOTES]: Addressed in https://github.com/EESSI/eessi-bot-software-layer/pull/232/commits/7320c1a31d6af51ad8f6923adda91057d58e6702

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@trz42 Do you see any reason not to go ahead and merge this, and tag the `0.2.0` release?

**Comment by trz42** [comment]
> @trz42 Do you see any reason not to go ahead and merge this, and tag the `0.2.0` release?

All fine. Let's move on.

**Review by boegel — APPROVED** [review]
